debian/control.top.in

```diff
@@ -2,7 +2,7 @@ Source: linuxcnc
 Section: misc
 Priority: optional
 Maintainer: Sebastian Kuzminsky <seb@highlab.com>
```

Comment:
The Maintainer should be that collective LinuxCNC address, I suggest, like you have it for mesaflash.

Reply:
We've historically elected a release manager for each release cycle and put them there, but I like using the group address instead.

Reply:
The Maintainer gets an email when there is a bug reported and also when a new version is uploaded. I do not know enough about how that email address is used, but I can say that it is not too much traffic. Just be aware that every email you place as Maintainer or Uploader is out in the public.